Source/IGListAdapter.m

```diff
@@ -308,6 +308,16 @@ - (NSUInteger)sectionForSectionController:(IGListSectionController <IGListSectio
     return [self.sectionMap sectionControllerForObject:object];
 }
 
+- (id)objectForSectionController:(IGListSectionController <IGListSectionType> *)sectionController {
+    IGAssertMainThread();
+    IGParameterAssert(sectionController != nil);
+    
+    NSUInteger section = [self.sectionMap sectionForSectionController:sectionController];
+    IGAssert(section != NSNotFound, @"Section nil for section controller %@", sectionController);
```

Comment:
Hmm if this assert will be too aggressive. Sometimes weird things happen and its probably ok to return `nil` in that case (for instance, the `-[IGListSectionMap objectForSection:]` method actually returns `nil` if the section is OOB).

I think we should probably omit this assert.